server/src/workflows/mail/sendReminderMailToUpdateContents/sendReminderMailToUpdateContents.ts

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-console */
+import { RequestFromClient, Res } from "../../../types/interface";
+import logger from "../../../logger";
+import { getPublishedDispositifWithMainSponsor } from "../../../modules/dispositif/dispositif.repository";
+import { getUserById } from "../../../modules/users/users.repository";
+import { checkCronAuthorization } from "../../../libs/checkAuthorizations";
+import { filterDispositifsForUpdateReminders } from "../../../modules/dispositif/dispositif.adapter";
+import { sendUpdateReminderMailService } from "../../../modules/mail/mail.service";
+// import { isTitreInformatifObject } from "../../../types/typeguards";
+
+export const sendReminderMailToUpdateContents = async (
+  req: RequestFromClient<{ cronToken: string }>,
+  res: Res
+) => {
+  try {
+    logger.info("[sendReminderMailToUpdateContents] received");
+
+    checkCronAuthorization(req.body.query && req.body.query.cronToken);
+
+    const dispositifs = await getPublishedDispositifWithMainSponsor();
+    logger.info(
+      `[sendReminderMailToUpdateContents] ${dispositifs.length} dispositifs find`
+    );
+
+    logger.info(
+      `[sendReminderMailToUpdateContents] ${dispositifs[0]} dispositifs find`
+    );
+
+    const nbDaysBeforeReminder = 90;
+
+    const filteredDispositifs = filterDispositifsForUpdateReminders(
+      //@ts-ignore
+      dispositifs,
+      nbDaysBeforeReminder
+    );
+
+    logger.info(
+      `[sendReminderMailToUpdateContents] find ${filteredDispositifs.length} reminders to send`
+    );
+
+    filteredDispositifs.forEach((dispositif) => {
+      if (dispositif.mainSponsor) {
```

Comment:
ce serait bien de mettre un try catch pour chaque element de la boucle for, si il y a un pb avec un dispositif ou un user on n'a quand meme envie denvoyer aux autre users /dispositifs